app/components/new-signup/input.hbs

```diff
@@ -16,13 +16,24 @@
   </div>
 
   <div class='signup-action__btn'>
-    <Button
-      @data-test-btn='signup'
-      @onClick={{@onClick}}
-      @disabled={{@isButtonDisabled}}
-      @isLoading={{@isLoading}}
-    >
-      {{if (eq @currentStep 'username') 'Submit' 'Next'}}
-    </Button>
+    {{#if @dev}}
+      <Button
+        @data-test-btn='signup'
+        @onClick={{@onClick}}
+        @disabled={{@isButtonDisabled}}
+        @isLoading={{@isLoading}}
+      >
+        Next
+      </Button>
+    {{else}}
+      <Button
+        @data-test-btn='signup'
+        @onClick={{@onClick}}
+        @disabled={{@isButtonDisabled}}
+        @isLoading={{@isLoading}}
+      >
+        {{if (eq @currentStep 'username') 'Submit' 'Next'}}
+      </Button>
+    {{/if}}
```

Comment:
Can we write the whole dev condition inside the button?

Reply:
Done can you please check my latest commit